requirements.txt

```diff
@@ -0,0 +1,37 @@
+bsddb3==6.2.4
+certifi==2017.4.17
+chardet==3.0.4
+cymem==1.31.2
+cytoolz==0.8.2
+dill==0.2.6
+docopt==0.6.2
+en-core-web-sm==1.2.0
+ftfy==4.4.3
+fuzzywuzzy==0.15.0
+html5lib==0.999999999
+idna==2.5
+munkres==1.0.12
+murmurhash==0.26.4
+nltk==3.2.4
+num2words==0.5.4
+numpy==1.13.0
+pathlib==1.0.1
+plac==0.9.6
+preshed==1.0.0
+regex==2017.4.5
+requests==2.18.1
+scikit-learn==0.18.2
+scipy==0.19.1
+six==1.10.0
+sklearn==0.0
+spacy==1.8.2
```

Comment:
There's a [new version of spacy](https://spacy.io/usage/v2) out (2.x).
I think it would make sense to try and use that one - I think it specifically aims at improving NER and parsing.
It might require some small code changes on our side ([see here](https://spacy.io/usage/v2#migrating)) but I think it might be worth it to do it now since you're dealing with spacy now anyway.

Reply:
done!